styles/globals.css

```diff
@@ -33,26 +41,44 @@ body{
 }
 
 .nav {
- padding: 15px;
+ padding: 1rem;
  list-style-type: none;
  text-align: center;
 }
 
 .nav-list-container{
   display:flex;
-  padding: 3px;
+  padding: 0.188rem;
 }
 
 .nav li {
-  padding: 0 20px;
+  padding: 0 1.25rem;
   cursor: pointer;
 } 
 .nav li:hover {
-color: #B01E68;
+color: var(--color-tertiary);
 text-decoration: none;
 }
 
+.poi-container{
```

Comment:
[sand] I dislike the name `poi-container`. I like to be specific with my names so we know exactly what we are editing

Reply:
Fixed this name and another